designer/server/src/test/scala/pl/touk/nussknacker/ui/api/ScenarioActivityApiHttpServiceBusinessSpec.scala

```diff
@@ -389,6 +399,322 @@ class ScenarioActivityApiHttpServiceBusinessSpec
           )
         )
     }
+    // todo: add test, that will verify limiting the number of scheduling-related activities returned by the API
```

Comment:
This test in todo is currently extremely hard to add - we have no configuration and no integration test with periodic DM. When I tried addint it, it is replaced somewhere deep in the code to "mockable" deployment manager.